SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -250,28 +250,28 @@ RealKills = 1
 # Во время боя в левой части экрана будет отображаться интерактивная панель быстрого доступа к наиболее часто используемым заклинаниям.
 # Die linke Seite des Kampf-Fensters zeigt eine interaktive Tafel mit Ihren festgesetzten Schnellzaubersprüchen an und erlaubt Anpassung.
 # Το αριστερό τμήμα της οθόνης μάχης εμφανίζει έναν πίνακα με τα γρήγορα ξόρκια που έχετε επιλέξει και επιτρέπει εξατομίκευση.
-QuickSpellPanel = 1
+QuickSpellPanel = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.